docs/docs/getting-started-aws/index.md

```diff
@@ -0,0 +1,140 @@
+3. Choose Create New Access Key. You can have a maximum of two access keys (active or inactive) at a time.
+4. Choose Download Key File to save the access key ID and secret access key to a .csv file on your computer. After you close the dialog box, you can't retrieve this secret access key again.
+5. To disable an access key, choose Make Inactive. AWS denies requests signed with inactive access keys. To re-enable the key, choose Make Active.
+6. To delete an access key, choose Delete. To confirm that the access key was deleted, look for Deleted in the Status column.
+
+> Before you delete an access key, make sure it is no longer in use. You can't recover a deleted access key.
+
+***
+
+## Create your Stack <a name="create-stack"></a>
+
+***[What is a Stack?](/docs/what-is-a-stack)***
+
+To create your Stack:
+
+1. [Login to Koding](https://koding.com/Teams/Select)
+2. Go to your [Dashboard](https://relepic.koding.com/Home) OR click [Stacks](https://relepic.koding.com/Home/Stacks) from your sidebar.
+3. Either way, you'll default to the Stacks screen, so from that screen click [New Stack](https://relepic.koding.com/Stack-Editor/New)
+4. Use the Wizard to create and  populate your Stack with some defaults, make sure you select Amazon Web Services as provider
+
+Your Stack will be created and will look something like this:
+
+```
+# Here is your stack preview
+# You can make advanced changes like modifying your VM,
+# installing packages, and running shell commands.
+
+provider:
+  aws:
+    access_key: '${var.aws_access_key}'
+    secret_key: '${var.aws_secret_key}'
+resource:
+  aws_instance:
+    example_1:
+      instance_type: t2.nano
+      ami: ''
+      tags:
+        Name: '${var.koding_user_username}-${var.koding_group_slug}'
+      user_data: |-
+        export DEBIAN_FRONTEND=noninteractive
+        apt-get update -y
+        apt-get -y install ruby nginx
+    example_2:
+      instance_type: t2.nano
+      ami: ''
+      tags:
+        Name: '${var.koding_user_username}-${var.koding_group_slug}'
+      user_data: |-
+        export DEBIAN_FRONTEND=noninteractive
+        apt-get update -y
+        apt-get -y install mysql-server
+```
+Make whatever changes you'd like to this script. Our [Stack Reference for AWS](https://www.terraform.io/docs/providers/aws/index.html) will help you see what is possible.
+
+> You won't need to worry about the aws_*_key variables. These will be automatically populated when you set up credentials in the next step. 
+
+***
+
+## Set your Credentials <a name="set-creds"></a>
+
+To add your AWS credentials:
+
+1. From your Edit Stack screen (if you've just created the Stack, you will be on that screen by default) click on the Credentials tab
+2. On the bottom right of the Credentials screen, click Create New
+3. Name your Credentials whatever you like (pick something that will help you recognize them at a glance)
+4. Paste in the ID and Key that you obtained [earlier](#create-key)
```

Comment:
Be more specific about Access Key and Secret Key, etc. and which step they obtained them from.  Reference the .csv file they downloaded.